packages/ckeditor5-language/src/textpartlanguageui.ts

```diff
@@ -11,6 +11,9 @@ import { Plugin } from 'ckeditor5/src/core';
 import { Model, createDropdown, addListToDropdown, type ListDropdownItemDefinition } from 'ckeditor5/src/ui';
 import { Collection } from 'ckeditor5/src/utils';
 import { stringifyLanguageAttribute } from './utils';
+import type TextPartLanguageCommand from './textpartlanguagecommand';
+
+import './textpartlanguageconfig';
```

Comment:
This is already imported in `packages/ckeditor5-language/src/textpartlanguageediting.ts`

Reply:
Yes, it is.
But `textpartlanguageediting` is not imported by this file. The goal of this PR is to make sure that every file which depends on module augmentation (which happens on line 39 here) imports the config.